Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml

```
@@ -0,0 +1,64 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: OpenPhish_v2
+  version: -1
+configuration:
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Database refresh interval (hours)
+  name: fetchIntervalHours
+  defaultvalue: 1
+  type: 0
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+description: OpenPhish uses proprietary Artificial Intelligence algorithms to automatically
+  identify zero-day phishing sites and provide comprehensive, actionable, real-time
+  threat intelligence.
+name: OpenPhish_v2
+display: OpenPhish v2
+defaultEnabled: false
+script:
+  dockerimage: demisto/python3:3.8.5.11789
+  commands:
+    - name: url
+      arguments:
+        - name: url
+          required: true
+          default: true
+          description: URL to check
+      outputs:
+        - contextPath: URL.Data
+          description: Bad URLs found
+        - contextPath: URL.Malicious.Vendor
+          description: For malicious URLs, the vendor that made the decision
+        - contextPath: URL.Malicious.Description
+          description: For malicious URLs, the reason for the vendor to make the decision
+        - contextPath: DBotScore.Indicator
+          description: The indicator we tested
+        - contextPath: DBotScore.Type
+          description: The type of the indicator
+        - contextPath: DBotScore.Vendor
+          description: Vendor used to calculate the score
+        - contextPath: DBotScore.Score
+          description: The actual score
```

https://xsoar.pan.dev/docs/integrations/context-standards#url
https://xsoar.pan.dev/docs/integrations/context-standards#dbot-score
I'm not sure this follows the standards, please review all outputs.
```suggestion
      outputs:
        - contextPath: URL.Data
          description: The URL.
        - contextPath: URL.Malicious.Vendor
          description: The vendor reporting the URL as malicious.
        - contextPath: URL.Malicious.Description
          description: For malicious URLs, the reason for the vendor to make the decision
        - contextPath: DBotScore.Indicator
          description: The indicator we tested
        - contextPath: DBotScore.Type
          description: The type of the indicator
        - contextPath: DBotScore.Vendor
          description: The vendor used to calculate the score.
        - contextPath: DBotScore.Score
          description: The actual score
```